Sources/5GUIs/Model/FileDetectionState.swift

```diff
@@ -213,6 +213,7 @@ final class FileDetectionState: ObservableObject {
       make(.catalyst, .catalyst),
       make(.swiftui,  .swiftUI),
       .init(config: .phone, state: isPhone),
+      make(.qt,  .qt),
```

Comment:
I'd like to drop this one line, all else is cool

Reply:
Removed from FakeStepConfig as well.